infinispan-remote/connect-to-infinispan-server/src/main/java/org/infinispan/tutorial/simple/connect/Infinispan.java

```diff
@@ -0,0 +1,62 @@
+package org.infinispan.tutorial.simple.connect;
+
+import org.infinispan.client.hotrod.RemoteCacheManager;
+import org.infinispan.client.hotrod.configuration.ClientIntelligence;
+import org.infinispan.client.hotrod.configuration.ConfigurationBuilder;
+import org.infinispan.client.hotrod.impl.ConfigurationProperties;
+
+public class Infinispan {
+
+   public static final String TUTORIAL_CACHE_NAME = "test";
+   public static final String TUTORIAL_CACHE_CONFIG = "<distributed-cache name=\"CACHE_NAME\">"
+         + "<encoding media-type=\"application/x-protostream\"/>"
+         + "</distributed-cache>";
```

Comment:
I like having a common class for the server connection so that the other tutorials don't have to show the same thing over and over. You can focus on capabilities instead of the connection. 

For the cache configuration, I'd prefer if we could put them in external XML files and reference them instead of adding the XML to Java as String. IMO it'd be nicer to keep XML and Java separate.

Reply:
We can do that definetely